presto-accumulo/src/test/java/com/facebook/presto/accumulo/serializers/AbstractTestAccumuloRowSerializer.java

```diff
@@ -99,16 +99,15 @@ public void testBoolean()
     public void testDate()
             throws Exception
     {
+        Date expected = new Date(System.currentTimeMillis());
```

Comment:
Use a fixed date here so that it's clear the test doesn't depend on the current time:
```java
Date expected = new Date(new DateTime(2001, 2, 3, 4, 5, 6, DateTimeZone.UTC).getMillis());